### PR TITLE
Update secure storage to document entitelments

### PR DESCRIPTION
**Pull request by jamesmontemagno**

### CODE REVIEW

**Review by jtaubensee — APPROVED** [review]

**Review by mattleibow — APPROVED** [review]

File: docs/essentials/secure-storage.md [hunk 1]
```diff
@@ -12,6 +12,28 @@ ms.date: 05/04/2018
 
 The **SecureStorage** class helps securely store simple key/value pairs.
 
+## Getting Started
+
+To access the **SecureStorage** functionality the following platform specific setup is required.
```
Comment (bradumbaugh) [inline docs/essentials/secure-storage.md]: - "functionality,"
- "platform-specific"
- "required:"

File: docs/essentials/secure-storage.md [hunk 2]
```diff
@@ -12,6 +12,28 @@ ms.date: 05/04/2018
...
+
+# [Android](#tab/android)
+
+No additional setup required.
+
+# [iOS](#tab/ios)
+
+When developing on the iOS Simulator it is required to enable the Keychain entitlement and add a Keychain group with the application's package identifier.
```
Comment (bradumbaugh) [inline docs/essentials/secure-storage.md]: - Is this necessary only for the simulator? Or for physical devices as well?
- "When developing on the iOS simulator, enable the **Keychain** entitlement and add a keychain access group for the application's bundle identifier."

File: docs/essentials/secure-storage.md [hunk 3]
```diff
@@ -12,6 +12,28 @@ ms.date: 05/04/2018
...
+
+Open the Entitlements.plist in the iOS project and find the Keychain entitlement and enable it. This will automatically add the application's identifier as a group.
```
Comment (bradumbaugh) [inline docs/essentials/secure-storage.md]: - "**Entitlements.plist**"
- "**Keychain** entitlement"

File: docs/essentials/secure-storage.md [hunk 4]
```diff
@@ -12,6 +12,28 @@ ms.date: 05/04/2018
...
+
+In the project properties under iOS Bundle Signing set the **Entitlements.plist** in the Custom Entitlements.
```
Comment (bradumbaugh) [inline docs/essentials/secure-storage.md]: - "properties, under **iOS Bundle Signing**"
- "set **Custom Entitlements** to **Entitlements.plist**."

**Review by bradumbaugh — CHANGES_REQUESTED** [review]
Looks good, just a few minor grammar/style things

File: docs/essentials/secure-storage.md (hunk 2 above)
Comment (jamesmontemagno) [reply docs/essentials/secure-storage.md]: From my understanding only simulator.. :( as i have it working on phone jsut fine.. @jtaubensee @mattleibow

**Comment by jamesmontemagno** [comment]
should be cleaned up now

**Review by bradumbaugh — APPROVED** [review]

**Pull request merged**